### PR TITLE
1905-10-23

### PR DESCRIPTION
**Pull request by alexissharpe07**
Mostly grammatical or editing errors. Fixing organization, datelines, checking names and places. Everything that was given was good, except a missing column from a page.

### CODE REVIEW

File: 1905-10-23.xml [hunk 1]
```diff
@@ -29,11 +30,11 @@
     </teiHeader>
     <text>
         <body>
-            <pb n="1"/>
+            <pb n="1"></pb>
```
Comment (whanley) [inline 1905-10-23.xml]: this should read `<pb n="1"/>`

**Comment by huoyanzhizhou** [comment]
In 1906-10-31, there are several small item div(s) outside of page div(s), please check your outline window to see this. Also, please use features, see: https://dig-eg-gaz.github.io/contents/features/

**Comment by whanley** [comment]
Please complete verification and address the comments on Deliverables 1 and 2.

There are some real problems, for example on page 6. You have not broken the material into items, and you have not used templates for a bunch of material that is in tables.

Ask if you are confused.

**Comment by whanley** [comment]
I'm closing this pull request, because there's a more recent pull request.

**Pull request closed without merging**